spark/v3.4/spark-runtime/src/integration/java/org/apache/iceberg/spark/SmokeTest.java

```diff
@@ -44,7 +45,7 @@ public void dropTable() {
   // Run through our Doc's Getting Started Example
   // TODO Update doc example so that it can actually be run, modifications were required for this
   // test suite to run
-  @Test
+  @Ignore
```

Comment:
Is this needed?

Reply:
This is not needed. 
Currently I set Comet reader as default to make sure all the tests will pass with Comet reader. I think I have to skip this test for Comet reader. Once I change back Iceberg reader as default, I will remove this change.

Reply:
Yeah, we will have to use the built-in reader by default.